### ISSUE TITLE
Upgrade to Solr 4

### ISSUE THREAD

**Issue by seancoyne** [jeffcoughlin/farcrysolrpro#53]

**Comment by tHeSmUrF** [comment]
Upgrade to Solr 5 :)

**Comment by seancoyne** [comment]
Yeah if we were to upgrade it would be to the current version.  That said, it is unlikely an upgrade will happen any time soon.  We haven't had need for any additional features not available in 3.5 and we haven't had the time to upgrade Solr on our own (though I did take a quick stab at it about a year ago, but did not get very far).  That said, if anyone wants to sponsor development or provide pull requests we would certainly be open to that.
